icon/parse

```diff
@@ -43,9 +43,9 @@ OMNI_LATEST=false
 CONTINUE_TEST=false
 
 # ICON related variables
-ICON_MAIN_REPO="git@github.com:C2SM-RCM/icon-eniac.git"
-ICON_REP="icon-eniac"
-STANDALONE_GIT_BRANCH=""
+ICON_MAIN_REPO="git@gitlab.dkrz.de:icon/icon.git"
+ICON_REP="icon-clone"
+STANDALONE_GIT_BRANCH="icon-2.6.2-rc"
```

Comment:
You should not set it here unless you want it to be your default branch. You should use the `-b` arg in the script to specify your branch in the Jenkins plan ...

Reply:
You are right, I set it back to empty.